File: sdk-element-standard/src/main/resources/archetype-resources/element/src/main/java/guice/MyGameModule.java
#set( $symbol_pound = '#' )
#set( $symbol_dollar = '$' )
#set( $symbol_escape = '\' )
package ${package}.guice;

import com.google.inject.PrivateModule;
import ${package}.service.GreetingService;
import ${package}.service.GreetingServiceImpl;

public class MyGameModule extends PrivateModule {

    @Override
    protected void configure() {

        bind(GreetingService.class).to(GreetingServiceImpl.class);

        expose(GreetingService.class);
    }
}

File: sdk-element-standard-kt/src/main/resources/archetype-resources/element/src/main/kotlin/__packageInPathFormat__/guice/MyGameModule.kt
package ${package}.guice

import com.google.inject.PrivateModule
import ${package}.service.GreetingService
import ${package}.service.GreetingServiceImpl

class MyGameModule : PrivateModule() {

    override fun configure() {
        bind(GreetingService::class.java).to(GreetingServiceImpl::class.java)
        expose(GreetingService::class.java)
    }

}
